evap/contributor/templates/contributor_index.html

```diff
@@ -52,7 +52,7 @@
                     </thead>
                     <tbody>
                     {% for course in semester.courses %}
-                        <tr>
+                        <tr {% if course|is_user_editor_or_delegate:user and course.state == 'prepared' %}class="hover-edit" data-url="{% url 'contributor:course_edit' course.id %}"{% elif course.state == 'published' %}class="hover-results" data-url="{% url 'results:course_detail' semester.id course.id %}"{% else %}class="nohover"{% endif %}>
```

Comment:
If a course is in evaluation, we can not edit it nor see its results. The column is not hilighted when hovering it. Maybe, we should add a hover effect which, when clicked, does the same as the last button (preview).

Reply:
For the record: When the default action is to edit the course, I don't like that the edit button is hilighted when I hover the preview button. We talked about this, you said its too hard to do it properly and I don't care that much, but if we find a solution in the future that would be nice.

Reply:
when a contributor already approved a course there are two available buttons: one for viewing the disabled form, another one for the preview. none of them is any "default option". because of that i think it would be best not to include any other functionality into the line links besides of editing and viewing results because it might become confusing.

Reply:
not applying the hover effect to the other button would be nice but seems to be possible only with additional js-code. would you like to open another issue for that?

Reply:
Hmm, with our current database, I don't get a button for viewing the disabled form when the course is already in evaluation - or maybe there is something different about this course? Am I missing anything?
![screenshot](https://user-images.githubusercontent.com/13838962/34799313-a79a9df4-f65f-11e7-8bcf-80910ae46fbb.png)

Yes, I'll create another issue for that :)

Reply:
the button was and is only available in the states `editor_approved` and `approved`. i didn't change that.